readme.txt

```diff
@@ -44,7 +44,7 @@ The full version of Sugar Calendar has several [premium license levels](https://
 
 - Manage recurring/repeating events
 - Duplicate an event with a single click
-- Access [add-ons](https://sugarcalendar.com/free-add-ons/) for integration with [Google Maps](https://sugarcalendar.com/downloads/google-maps/?utm_medium=referral&utm_source=website&utm_campaign=wporg-scl-plugin-page), [Gravity Forms](https://sugarcalendar.com/downloads/gravity-forms-form-embed/?utm_medium=referral&utm_source=website&utm_campaign=wporg-scl-plugin-page), and [Ninja Forms](https://sugarcalendar.com/downloads/ninja-forms-form-embed/?utm_medium=referral&utm_source=website&utm_campaign=wporg-scl-plugin-page)
```

Comment:
I see that I missed this single `[add-ons]` instance in my first pass commit.

If the team decides to keep the URLs/slugs as `add-on` (for SEO or other reasons), this one thing will need to be redone and merged. Not a big deal, just noting it for the future! 🙏